code/game/machinery/kitchen/processor.dm

```diff
@@ -216,6 +216,8 @@
 	return
 
 /obj/machinery/processor/attack_hand(var/mob/user as mob)
+	if(!ishuman(user) && !isrobot(user)) //No ghosts or mice putting people into the sleeper
```

Comment:
Wait hold on
Why not just override `attack_ghost()`?

I hate how you can't edit comments on mobile